package.json

```diff
@@ -95,7 +95,7 @@
       "eslint --cache --fix"
     ]
   },
-  "packageManager": "pnpm@9.15.4",
+  "packageManager": "pnpm@9.15.5+sha512.845196026aab1cc3f098a0474b64dfbab2afe7a1b4e91dd86895d8e4aa32a7a6d03049e2d0ad770bbe4de023a7122fb68c1a1d6e0d033c7076085f9d5d4800d4",
```

Comment:
I added the hash here to workaround https://github.com/nodejs/corepack/issues/612. We can upgrade corepack as well, but this approach is more friendly to contributors as it works with older versions of corepack (Only node 23 includes the latest corepack, and the users have to upgrade it).